### DISCUSSION TITLE
Unable to run wiki on Raspberry Pi 4B

### DISCUSSION THREAD

**Discussion by njacobs5074** [requarks/wiki#4801] [Error / Bug Report]
I'm struggling to get the Docker compose script at https://docs.requarks.io/install/docker to work.

I get an endless loop of this on stdout:
```
wiki_1  | #
wiki_1  | # Fatal error in , line 0
wiki_1  | # unreachable code
wiki_1  | #
wiki_1  | #
wiki_1  | #
wiki_1  | #FailureMessage Object: 0xbe9ad1fc
```

Here's some, hopefully relevant, information about my Pi:
```
$ docker --version
Docker version 20.10.12, build e91ed57

$ raspinfo
System Information
------------------

Raspberry Pi 4 Model B Rev 1.2
PRETTY_NAME="Raspbian GNU/Linux 10 (buster)"
NAME="Raspbian GNU/Linux"
VERSION_ID="10"
VERSION="10 (buster)"
...
```

Happy to share more info.

**Comment by jg3** [comment]
Hi!  We're having the same issue over here.  Hopefully someone can shed some light ...

**Reply by jg3** [reply]
I haven't been able to take it all the way to ground but [this discussion](https://github.com/Koenkk/zigbee2mqtt/issues/7662) seems to have similar symptoms and points to a conflict in the version of `libseccomp2` in Alpine and the Raspberry Pi.  Maybe that sheds some light?

**Comment by capttrousers** [comment]
I had this issue as well with a newer version of wikijs on another raspberrypi, but an older version was running fine on a different pi. 
The issue is due to the base node-alpine docker image used to build the wikijs docker image, and the MUSL library, as referenced by jg3 in that linked discussion. 

Both raspberry pi's were running raspbian Buster (v10) which is EOL as of fall 2022, so I upgraded one of the pi's to Bullseye (v11) using this guide: https://linuxhint.com/upgrade-raspberry-pi-os-buster-to-bullseye/ and the latest wikijs version of the docker image works.
